tests/observability/datasources.yaml

```diff
@@ -0,0 +1,94 @@
+$schema: ../../json_schemas/test_story.schema.yaml
```

Comment:
This is for `query`, so it should live in `tests/query/datasources.yml`, right?

Reply:
I will move datasource test story to /tests/default/query/. But do you think I should move observability to default ?  Cause as far as I know observability is an external service

Reply:
This is a bit new in this repo.

Folders like `default` and `snapshot` are there to have separate docker-compose.yml, not because something is external or internal. These are "test suites". So you should move into `tests/default/query` and `tests/default/observability`. 

If observability required some fancy setup, you could create a new folder for it and add a job to the GHA, but it is hopefully not necessary.